src/normalize_axes.cpp

```diff
@@ -5,114 +5,160 @@
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
-bool normalize_axes(operation& op, const std::vector<std::size_t>& lens)
+// different attributes
+// 1) use_input(default)/use_output
+// 2) use_rank(default)/use_len
+// 3) clip_min(default)/not_clip_min
+//   3.1) include_min(default)/exclude_min
+// 4) clip_max(default)/not_clip_max
+//   4.1) exclude_max(default)/include_max
```

Comment:
I wonder if we should use a enum instead of raw strings. We can probably change that in a later PR.

Reply:
Changed